packages/plugin-ga-events-forwarder-browser/src/ga-events-forwarder.ts

```
@@ -152,6 +159,7 @@ export const gaEventsForwarderPlugin = ({ measurementIds = [] }: Options = {}):
     } catch (e) {
       /* istanbul ignore next */
       logger?.error(e);
```

Can we just change it to logger.debug() instead so that we don't silently catch an error when url is invalid

---

> we don't silently catch an error when url is invalid

The thing that's different here is that this callback is invoked via monkeypatching of fetch and sendBeacon. It intercepts whatever URL was sent to that

```
globalScope.navigator.sendBeacon = new Proxy(globalScope.navigator.sendBeacon, {
  apply: (target: SendBeaconFn, thisArg: any, argArray: Parameters<SendBeaconFn>) => {
    // Intercepts request and attempt to send to Amplitude
    interceptBeacon.apply(thisArg, argArray);
    // Execute sendBeacon
    return target.apply(thisArg, argArray);
  },
});
```

So even if the URL is invalid, we don't really have any reason to be logging errors or debug, because it's not our error.